ui/src/layouts/storage.rs

```diff
@@ -0,0 +1,302 @@
+use std::time::Duration;
+
```

Comment:
Is this file renamed to Storage for any specific reason or just matching warp? I'm okay with whatever just want to be consistent across the app.